planner/core/casetest/rule/testdata/ordered_result_mode_suite_out.json

```diff
@@ -440,7 +440,8 @@
     "Cases": [
       {
         "Plan": [
-          "Batch_Point_Get_9 2.00 root table:thash handle:[1 200], keep order:true, desc:false"
+          "TableReader_10 2.00 root partition:p0,p1 data:TableRangeScan_9",
+          "└─TableRangeScan_9 2.00 cop[tikv] table:thash range:[1,1], [200,200], keep order:true, stats:pseudo"
```

Comment:
Is this a regression, or was it wrong before?

Reply:
It's wrong before. Can't support keepOrder + dynamic pruning.